challenges/models.py

```diff
@@ -4,9 +4,13 @@
 class Challenge(models.Model):
     name = models.CharField(max_length=100)
     description = models.TextField()
-    duration = models.IntegerField(help_text='how long it takes to complete challenge')
-    frequency = models.IntegerField(help_text='how often challenge is activated: 1/day, 1/week, 1/month, 1/year')
-    points = models.IntegerField(help_text='how many points are gained each time challenge is completed')
+    duration = models.IntegerField(help_text="how long it takes to complete challenge")
+    frequency = models.IntegerField(
```

Comment:
Jeżeli są tylko trzy opcje to może warto ogarnąć jakąś walidację do tego? 
Niekoniecznie w tej PRce, ale fajnie jakbyś zrobiła sobie na to issue.

Reply:
W kolejnym zadaniu- chodzi o to, żeby przy tworzeniu nowego Challenge była możliwość wybrania czasu trwania DURATION tylko z 3 wartości: 7/30/365?

Reply:
Ten komentarz dotyczy `frequency`. W szczególności odnosi się do jego help texta.

https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue

Reply:
OK, zapisane.